h2o-r/h2o-package/R/frame.R

```diff
@@ -1716,7 +1716,8 @@ h2o.tail <- function(x,n=6L,...) {
   if( n==0L ) head(x,n=0L)
   else {
     startidx <- max(1L, endidx - n + 1)
-    as.data.frame(.newExpr("rows",x,paste0("[",startidx-1,":",(endidx-startidx+1),"]")))
+    df = .newExpr("rows",x,paste0("[",startidx-1,":",(endidx-startidx+1),"]"))
+    head(.fetch.data(df,startidx-1),n)
```

Comment:
Yes, "head" seems redundant, h2o::fetch.data already returns an R data.frame

class(h2o::.fetch.data(XX, 1, 2))
> data.frame
is.data.frame(h2o::.fetch.data(XX, 1, 2))
> TRUE